src/WaveTrack.cpp

```diff
@@ -1759,7 +1759,7 @@ void WaveTrack::Join(double t0, double t1)
 -- Some prefix (maybe none) of the buffer is appended,
 and no content already flushed to disk is lost. */
 bool WaveTrack::Append(constSamplePtr buffer, sampleFormat format,
-                       size_t len, unsigned int stride /* = 1 */)
+   size_t len, unsigned int stride, sampleFormat)
```

Comment:
a new sampleFormat param was added, but it is not used? maybe it is a placeholder for future use?